src/views/editor/view.rs

```diff
@@ -892,13 +1004,23 @@ impl Widget for EditorView {
         // I expect that most/all of the paint functions could restrict themselves to only what is
         // within the active screen lines without issue.
         let screen_lines = ed.screen_lines.get_untracked();
-        EditorView::paint_cursor(cx, &ed, self.is_active.get_untracked(), &screen_lines);
+        EditorView::paint_cursor(
+            cx,
+            &ed,
+            self.is_active.get_untracked(),
+            &screen_lines,
+            &self.editor_view_style,
+        );
         let screen_lines = ed.screen_lines.get_untracked();
-        EditorView::paint_text(cx, &ed, viewport, &screen_lines);
-        EditorView::paint_scroll_bar(cx, &ed, viewport);
+        EditorView::paint_text(cx, &ed, viewport, &screen_lines, &self.editor_view_style);
+
+        // TODO: what is going on here? Why is this necessary
+        EditorView::paint_scroll_bar(cx, viewport, &self.editor_view_style);
```

Comment:
> (Possibly we could/should just remove the custom scrollbar painting code from floem-editor as Lapce has to write its own function on top for extra git map stuff anyway. Unsure)

I'll look at removing it would be more consistent with the rest of floem if it just used the one from the scroll view.